README.md

```diff
@@ -38,6 +38,7 @@ Therefore, for development purposes, you will need to set the source start time
 Pipeline | Schedule | Notes |
 --- | --- | --- |
 benchmark-timed-schedule.yaml | 3 AM every day | Deploys infrastructure required to carries out a performance test using eq-survey-runner-benchmark for which the outputs are stored in a GCS bucket. Once complete the infrastructure is torn down ready for the next scheduled test.
+stress-test.yaml | Manually triggered | Deploys infrastructure required to carry out a stress test of eq-questionnaire-runner. The outputs are stored in a GCS bucket.
```

Comment:
- This table needs to be updated now that the pipelines have moved into folders. We can probably drop the filename and just reference the folder name. 
- Is the stress test the same as the daily test other than the timing? If so, the description should probably just say exactly the same, or even "Same as daily-test"

Reply:
The only difference is that `Once complete the infrastructure is torn down ready for the next scheduled test.` doesn't happen.

Reply:
There is a destroy task..